contracts/src/v0.8/automation/test/v2_3_zksync/BaseTest.t.sol

```diff
@@ -81,9 +80,9 @@ contract BaseTest is Test {
     vm.startPrank(OWNER);
     linkToken = new LinkToken();
     linkToken.grantMintRole(OWNER);
-    usdToken18 = new ERC20Mock("MOCK_ERC20_18Decimals", "MOCK_ERC20_18Decimals", OWNER, 0);
```

Comment:
The most significant code change in this PR is the not-exposed ERC20Mock that was previously in vendor/oz now has to live in our own repo.